repository/h.json

```diff
@@ -621,6 +621,17 @@
 				}
 			]
 		},
+		{
+			"name": "Hippie Autocomplete",
+			"details": "https://github.com/Suor/sublime-hippie-autocomplete",
+			"labels": ["auto-complete"],
+			"releases": [
+				{
+					"sublime_text": ">4000",
```

Comment:
This should be `>=4050` (first public dev build) or `>=4107` (first public stable build) at least.

Reply:
Fixed